functions/handlers/interview.js

```diff
@@ -1,4 +1,18 @@
-// TODO: Get All Walkin Interviews - [interview]
+const { db } = require("../utils/admin");
```

Comment:
There si no walkin interview collection just interview with type "walking"

Reply:
the way this should be implemented is you search all the panels with walkin-interviews are enabled and get all the panels in return